lang/po/Firmware_sv.po

```diff
@@ -18,482 +18,482 @@ msgstr ""
-msgstr ""
+msgstr "Blackout inträffat. Aterställa utskr?"
 
 # MSG_CALIBRATING_HOME c=20
 #: ultralcd.cpp:8413
 msgid "Calibrating home"
-msgstr ""
+msgstr "Kalibrerar hem"
 
 # MSG_CALIBRATE_BED c=18
 #: ultralcd.cpp:5787
 msgid "Calibrate XYZ"
-msgstr ""
+msgstr "Kalibrerar XYZ"
 
 # MSG_HOMEYZ c=18
 #: messages.c:52
 msgid "Calibrate Z"
-msgstr ""
+msgstr "Kalibrera Z"
 
 # MSG_CALIBRATE_PINDA c=17
 #: ultralcd.cpp:4529
 msgid "Calibrate"
-msgstr ""
+msgstr "Kalibrerar"
 
 # MSG_CANCEL2 c=10
 #: messages.c:20
 msgid ">Cancel"
-msgstr ""
+msgstr ">Avbryt"
 
 # MSG_MOVE_CARRIAGE_TO_THE_TOP c=20 r=8
 #: ultralcd.cpp:3318
 msgid "Calibrating XYZ. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
-msgstr ""
+msgstr "Kalibrerar XYZ. Vrid vredet för att flytta Z-vagnarna upp till ändstoppen. Klicka när du är klar."
 
 # MSG_CALIBRATE_Z_AUTO c=20 r=2
 #: messages.c:21
 msgid "Calibrating Z"
-msgstr ""
+msgstr "Kalibrerar Z"
 
 # MSG_MOVE_CARRIAGE_TO_THE_TOP_Z c=20 r=8
 #: ultralcd.cpp:3318
 msgid "Calibrating Z. Rotate the knob to move the Z carriage up to the end stoppers. Click when done."
-msgstr ""
+msgstr "Kalibrerar Z. Vrid vredet för att flytta Z-slädarna upp till ändstoppen. Klicka när du är klar."
 
 # MSG_HOMEYZ_DONE c=20
 #: ultralcd.cpp:656
 msgid "Calibration done"
-msgstr ""
+msgstr "Kalibraring utförd"
 
 # MSG_MENU_CALIBRATION c=18
 #: messages.c:68
 msgid "Calibration"
-msgstr ""
+msgstr "Kalibrering"
 
 # MSG_SD_REMOVED c=20
 #: ultralcd.cpp:8860
 msgid "Card removed"
-msgstr ""
+msgstr "Kort borttaget"
 
 # MSG_CHECKING_FILE c=17
 #: ultralcd.cpp:8501
 msgid "Checking file"
-msgstr ""
+msgstr "Kontrollerar fil"
 
 # MSG_NOT_COLOR c=19
 #: ultralcd.cpp:2565
 msgid "Color not correct"
-msgstr ""
+msgstr "Färg ej korrekt"
 
 # MSG_COOLDOWN c=18
 #: messages.c:27
 msgid "Cooldown"
-msgstr ""
+msgstr "Kyla ner"
 
 # MSG_COPY_SEL_LANG c=20 r=3
 #: ultralcd.cpp:4435
 msgid "Copy selected language?"
-msgstr ""
+msgstr "Kopiera det valda spraket?"
 
 # MSG_CRASHDETECT c=13
 #: messages.c:30
 msgid "Crash det."
-msgstr ""
+msgstr "Krockdetekt."
 
 # MSG_CHOOSE_FIL_1ST_LAYERCAL c=20 r=7
 #: ultralcd.cpp:4842
 msgid "Choose a filament for the First Layer Calibration and select it in the on-screen menu."
-msgstr ""
+msgstr "Välj ett filament för första lagrets kalibrering och välj det i skärmmenyn."
 
 # MSG_CRASH_DETECTED c=20
 #: messages.c:29
 msgid "Crash detected."
-msgstr ""
+msgstr "Krock upptäckt."
 
 # MSG_CRASH_RESUME c=20 r=3
 #: Marlin_main.cpp:651
 msgid "Crash detected. Resume print?"
-msgstr ""
+msgstr "Krock upptäckt. Ateruppta utskrift?"
 
 # MSG_CRASH c=7
 #: messages.c:28
 msgid "Crash"
-msgstr ""
+msgstr "Krock"
 
 # MSG_CURRENT c=19
 #: ultralcd.cpp:5929
 msgid "Current"
-msgstr ""
+msgstr "Aktuell"
 
 # MSG_DATE c=17
 #: ultralcd.cpp:1985
 msgid "Date:"
-msgstr ""
+msgstr "Datum:"
 
 # MSG_COMMUNITY_MADE c=18
 #: messages.c:25
 msgid "Community made"
-msgstr ""
+msgstr "Allmänhetsgjord"
```

Comment:
I struggled with this one in Norwegian. I think I just left it as it was. Does not Allmänhetsgjord translate more to "the general public-done" not a specific community?